extensions/redis-client/runtime/src/etc/RedisCommandGenerator.java

```diff
@@ -0,0 +1,81 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
```

Comment:
Why is this file in `extensions/redis-client/runtime/src/etc/`

as opposed to where `extensions/redis-client/runtime/src/test/java/io/quarkus/redis/generator` just like https://github.com/quarkusio/quarkus/blob/5927a3b91bf0dd2dd87f3d026fdeea02545b63ff/extensions/redis-client/runtime/src/test/java/io/quarkus/redis/generator/RedisApiGenerator.java

Reply:
They are not doing the same thing.

The generator in `etc` is trying to generate the interface and data types for a specific group. It's a good first step, but not enough. I'm using it for the Javadoc. 

The generator in the `tests` is taking the complete reactive interface for a group and generates the synchronous interface, the abstract impl skeleton, the transactional interfaces, and the implementation skeleton....